### PR TITLE
1648 support log opt

### PR DESCRIPTION
**Pull request by mnowster**
Add in support for --log-opt as requested in issue https://github.com/docker/compose/issues/1648

### CODE REVIEW

File: docs/yml.md [hunk 1]
```diff
@@ -248,6 +248,14 @@ The default value is json-file.
     log_driver: "syslog"
     log_driver: "none"
 
+Specify logging options for the logging driver, as with the ``--log-opt`` option for docker run.
+
```
Comment (aanand) [inline docs/yml.md]: Is this sentence missing a "with `log_opt`"?

File: docs/yml.md [hunk 2]
```diff
@@ -248,6 +248,14 @@ The default value is json-file.
...
+Logging options are key value pairs. An example syslog options:
```
Comment (aanand) [inline docs/yml.md]: "example of"

**Comment by aanand** [comment]
ping @moxiegirl - docs look OK?

File: docs/yml.md [hunk 3]
```diff
@@ -248,6 +248,14 @@ The default value is json-file.
     log_driver: "syslog"
     log_driver: "none"
 
+Specify logging options with `log_opt` for the logging driver, as with the ``--log-opt`` option for docker run.
```
Comment (moxiegirl) [inline docs/yml.md]: `docker run` or Docker `run`

And `syslog`

Basically the rule is tics around anything that you type on a command line or see in a file system listing.

**Comment by mnowster** [comment]
Ah yes, good catch, I'd remembered for some of them to put in tics but missed those. Thank you for your feedback @moxiegirl :)

**Comment by moxiegirl** [comment]
Thank you @mnowster -- like how you test and write.  :-D

**Comment by mnowster** [comment]
@moxiegirl :sparkles: ah thank you, that's very kind :)

**Comment by moxiegirl** [comment]
...and look at (correct the code )... At some point, I hope to have the same luxury...:-P

**Comment by aanand** [comment]
LGTM

**Comment by mpareja** [comment]
Very useful stuff, can't wait to see it land in a release!

**Pull request merged**